Makefile

```diff
@@ -60,12 +65,16 @@ test/e2e-latest-image:
 	# Doesn't need cluster/prepare as it's done by operator-sdk. Uses a randomly generated namespace (instead of keycloak namespace) to support parallel test runs.
 	operator-sdk run local ./test/e2e --go-test-flags "-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count" --debug --verbose
 
-.PHONY: test/e2e-local-image cluster/prepare setup/operator-sdk
-test/e2e-local-image: cluster/prepare setup/operator-sdk
+.PHONY: test/e2e-local-image
+test/e2e-local-image: setup/operator-sdk
 	@echo Running e2e tests with a fresh built operator image in the cluster:
 	docker build . -t keycloak-operator:test
+	@echo Backing up operator.yaml
+	@cp deploy/operator.yaml deploy/operator.yaml_bckp
+	@echo Modifying operator.yaml
+	@sed -i 's/imagePullPolicy: Always/imagePullPolicy: Never/g' deploy/operator.yaml
```

Comment:
The image is not uploaded to any Registry so it cannot be pulled.

Reply:
Should we also change the image to **keycloak-operator:test** in **deploy/operator.yaml**?

Reply:
No, not permanently. By default it should use current master.

It's overridden only during testing and then reverted back to master.

Reply:
This is done transparently by Operator SDK's `--image` parameter.

Reply:
You can double check it in the test log – it's printed out from where the operator is deployed.

Reply:
All right. Then I have no further objections. Thanks @slaskawi!